.github/workflows/evm-benchmark.yml

```diff
@@ -0,0 +1,52 @@
+name: EVM benchmark
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - core/vm/**
+  workflow_dispatch:
+    inputs:
+      oldCommit:
+        description: "Old commit hash"
+        required: true
+        types: string
+      newCommit:
+        description: "New commit hash"
+        required: true
+        types: string
+
+concurrency:
+  group: benchmark-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  benchmark:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    env:
+      OLD_COMMIT: ${{ inputs.oldCommit }}
+      NEW_COMMIT: ${{ inputs.newCommit }}
+    steps:
+      - name: 'Checkout Repo'
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c #v3.3.0
+        with:
+          fetch-depth: 0
+
+      - name: 'Setup Go'
+        uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568 #v3.3.0
+        with:
+          go-version: '1.20.10'
+
+      - name: 'Run EVM benchmark'
+        run: |
+          go install golang.org/x/perf/cmd/benchstat@v0.0.0-20240604174448-3b48cf0e0164
+          cd core/vm
+          if [[ ${{ github.event_name }} = "pull_request" ]]; then
+            ../../script/benchstat.sh -c "go test -test.v -run=^$ -bench=BenchmarkEvm -count=10" \
```

Comment:
-count=10 make it input too ? maybe set default is 10

Reply:
This tool requires the benchmark to run at least 10 times to make the benchmark statistically significant

Reply:
ya I means, we can put it like input (optional) with default is 10, then defautly it still use 10, but we can adjust in case we need.

Reply:
like this

```
      go_version:
        description: 'The Go version to download (if necessary) and use.'
        required: false
        type: string
        default: '1.21.3'
```

Reply:
I just feel this is unnecessary, AFAIK, there is no scenario that we need to change that number.